src/config.ts

```diff
@@ -6,6 +6,7 @@ export default {
   applicationName: `CLI for Microsoft 365 v${app.packageJson().version}`,
   delimiter: 'm365\$',
   cliEntraAppId: process.env.CLIMICROSOFT365_ENTRAAPPID || process.env.CLIMICROSOFT365_AADAPPID || cliEntraAppId,
+  cliEnvEntraAppId: process.env.CLIMICROSOFT365_ENTRAAPPID || process.env.CLIMICROSOFT365_AADAPPID,
```

Comment:
This seems duplicate of the line above

Reply:
I needed to circumvent it defaulting to the pnp management shell Id... I only need the value if it's in an environment variable.

Reply:
Got it. Since we'll be no longer using the hardcoded value, do we still need this?